### ISSUE TITLE
parameter is_training when train a resnet_v1_50 from scratch and eval with the saved checkpoint file 

### ISSUE THREAD

**Issue by wx1111** [tensorflow/models#5667]
Please go to Stack Overflow for help and support:

http://stackoverflow.com/questions/tagged/tensorflow

Also, please understand that many of the models included in this repository are experimental and research-style code. If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **What is the top-level directory of the model you are using**:  models/research/slim
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: 
 no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: ubuntu16.04
- **TensorFlow installed from (source or binary)**: 
- **TensorFlow version (use command below)**: 1.8
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**: cuda9.0-cudnn7
- **GPU model and memory**: Tesla V100
- **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

 i trained a resnet_v1_50 model from scartch on imagenet dataset and then validated on the same dataset using the saved checkpoint files. while **the validation accuracy** was **alomst 0** when the is_parameter parameter was **False** in the eval_image_classifier.py and was **0.57692** when i changed the is_traing to **True**. the accuracy is acceptable since the training step is not finished. i think there is something wrong about the batch nomarlization layer.

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.
my train command : 
python train_image_classifier.py \
    --train_dir= \
    --dataset_name=imagenet \
    --dataset_split_name=train \
    --dataset_dir= \
    --model_name=resnet_v1_50 \
    --learning_rate = 0.1 \
    --batch_size =128
the eval command:
     python eval_image_classifier.py
     --dataset_name=imagenet \
     --dataset_split_name=validation \
     --model_name= resnet_v1_50 \
     --preprocessing_name=vgg\
      --labels_offset=0
and i chenged this paremeter in the eval_image_classifier.py
![image](https://user-images.githubusercontent.com/39900603/47896565-14868680-dea9-11e8-96bc-30a0ed2bc03e.png)

@tensorflow-jenkins
